### ISSUE TITLE
Create a zip for the website

### ISSUE THREAD

**Issue by johanvdw** [FOSDEM/pretalx-integration#20]
Currently the website build will rsync the exported resources from the pretalx server.

While this is ok for most users, if external users want to do a test-build of the website they have no way to do it.

Perhaps we should build a zipfile with these resources whenever we release.

**Comment by kergon** [comment]
On Fri, Dec 08, 2023 at 01:45:10PM -0800, Johan Van de Wauw wrote:
>    Perhaps we should build a zipfile with these resources whenever we
>    release.

Or consider a dummy zipfile/tarball that has one small genuine file of each
type (pdf etc.) and a symlink to it from each named file that exists in the
real contents.

Testers don't need copies of the actual files (which would be enormous)
so just replace each pdf with a symlink to a tiny standard pdf file 
and do the same for every other attachment filetype.

Alasdair

**Comment by johanvdw** [comment]
Good idea we can probably symlink to the files under sample (and add a few if needed).

**Comment by johanvdw** [comment]
This was tackled by having a --fake-image option, which replaces all images with 1 pixel fake things in the export.
